index.html

```diff
@@ -181,6 +181,30 @@ <h2 class="langFrameAndSkillsTitle">Skills</h2>
         </ul>
       </section>
     </article>
+    <article id="contact-form-container">
+      <header>
+        <h2 class="headline">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      </header>
+      <section id="form-background">
+        <form id="contact-form" action="https://formspree.io/f/mzbwkgyl" method="POST">
+          <input class="input-fields" type="text" name="name" id="name" max="30" placeholder="Full Name" required>
+          <input class="input-fields" type="email" name="email" id="email" placeholder="Email Address" required>
+          <textarea class="input-textarea" name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
```

Comment:
Good job! but it's required to limit the max length of the message field to 500 characters. Kindly make sure to fix it.